bot/cogs/weather.py

```diff
@@ -0,0 +1,105 @@
+"""Commands for the weather module."""
+import discord
+from discord.ext import commands
+import aiohttp
+from bot.settings import WEATHER_TOKEN
+
+
+class Weather(commands.Cog, name='weather'):
+    """commands for the weather finder."""
+
+    def __init__(self, bot: discord.ext.commands.bot.Bot):
+        self.bot = bot
+
+    @commands.command(
+        brief="Takes in a city name and returns the weather for that location")
+    async def weather(self, ctx: discord.ext.commands.context.Context, *, args: str) -> None:
+        """Weather command takes in a city name and sends embed."""
+        embed = discord.Embed(title=f"Weather in {args}")
+        url = "http://api.openweathermap.org/data/2.5/weather?q="
+        args = args.replace(' ', '%20')
+        imperial_url = f"{url}{args}&appid={WEATHER_TOKEN}&units=imperial"
+        metric_url = f"{url}{args}&appid={WEATHER_TOKEN}&units=metric"
+        async with aiohttp.ClientSession() as session:
+            async with session.get(metric_url) as resp:
+                metric = await resp.json()
+        async with aiohttp.ClientSession() as session:
+            async with session.get(imperial_url) as resp:
+                data = await resp.json()
```

Comment:
This requires logical line breaks to group and format what is happening.

you should also be able to use one session for both get requests here, so you can remove one `async with aiohttp.ClientSession() as session:` and next the other get request context.